stations.csv

```diff
@@ -70490,3 +70490,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74631;Bologna Centrale AV;bologna-centrale-av;;;44.5061410000;11.3434060000;10456;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S05046;;;;f;;f;;f;;f;;f;;f;f;;;;Bolonia;Bologne;;;;;;;ボローニャ;볼로냐;Bolonia;Bolonha;Болонья;;Bolonya;博洛尼亚;urn:trainline:public:nloc:csv74631;
 74632;Brescia Violino;brescia-violino;;;45.54553750281283;10.168658833685743;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8317451;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74632;
 74633;Minusio;minusio;;;46.17404650703084;8.819574802968333;;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8505417;t;;f;;f;;f;8505417;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74633;
+74634;Catania (p.zza risorgimento);catania-pp.zza-risorgimento;;;37.50155099221305;15.068981718421918;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74634;
```

Comment:
There's an extra semi-colon in your line (76 semi-colons vs 75 in other lines). It seems to be situated near the `trenitalia_is_enabled` column.

Reply:
Pushed changes. Thanks for reviewing

Reply:
Could you please rename the station into "Catania (Piazza Risorgimento)" (with capital letters)?

Reply:
Also, the station should be linked to the parent station 22160 (Catania)